src/profile/components/__mocks__/loadingApp.mockStore.js

```diff
@@ -3,6 +3,10 @@ module.exports = {
     userId: 9,
     username: 'staff'
   },
+  configuration: {
```

Comment:
Ideally these "mockStore" files should probably be replaced with factories... this is pretty un-DRY.

Reply:
It'd help make sure they don't get out of date by having something like:

factory.create('configuration'), where 'configuration' was a kind of known object with a definition in a factory file somewhere.  All tests that need a configuration object would get it from the same factory, so they'd all get updated if it changes, basically.  Same would go for any other object in there, like userAccount.